integration-test/src/main/java/org/apache/iotdb/it/framework/IoTDBTestRunnerWithParameters.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.it.framework;
+
+import org.apache.iotdb.it.env.EnvFactory;
+
+import org.junit.runner.Description;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.InitializationError;
+import org.junit.runners.parameterized.BlockJUnit4ClassRunnerWithParameters;
+import org.junit.runners.parameterized.TestWithParameters;
+import org.slf4j.Logger;
+
+public class IoTDBTestRunnerWithParameters extends BlockJUnit4ClassRunnerWithParameters {
```

Comment:
Can this runner be merged to `IoTDBTestRunner` to simplify the framework usage, which means that 
`IoTDBTestRunner` can be used with an optional parameter.

Reply:
It seems difficult to implement because Parameize requires the runner to be a subclass of `BlockJUnit4ClassRunnerWithParameters`, but `IoTDBRunner` is a subclass of `BlockJUnit4ClassRunner`.